src/Mod/Arch/archadd.py

```diff
@@ -0,0 +1,31 @@
+# ***************************************************************************
+# *   Copyright (c) 2020 Bernd Hahnebach <bernd@bimstatik.org>              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
+__title__ = "FreeCAD arch add objects methods"
+__author__ = "Bernd Hahnebach"
+__url__ = "http://www.freecadweb.org"
+
+from archmake.make_base_rebar import make_base_rebar as BaseRebar
```

Comment:
I'm not sure how this is supposed to work. Is `BaseRebar` a class? But `make_base_rebar` is a function? You are importing a function to look like a class?

Reply:
I don not know what is best here. In FEM we have all makes in one module. Furthermore I do not like the make. I would like to change this to add or even nothing just the obj. name. But this is a whole other topic not to be disscussed here. I will move the archadd back to my add on. Thus I can continue to use it.